webapp/api/__init__.py

```diff
@@ -0,0 +1 @@
+from . import requests # noqa
```

Comment:
Do you really need this? Wouldn't simply the existence of the `__init__.py` be enough?